assets/js/util.js

```diff
@@ -242,8 +243,26 @@ function synchronizeTextareaHeights() {
     $('#translatedText').css('height', originalTextScrollHeight + 'px');
 }
 
-/*:: export {synchronizeTextareaHeights, modeEnabled, ajaxSend, ajaxComplete, filterLangList, onlyUnique,
-    SPACE_KEY_CODE, ENTER_KEY_CODE, HTTP_OK_CODE, HTTP_BAD_REQUEST_CODE, XHR_LOADING, XHR_DONE} */
+function apyCall(requestObject, endpoint) {
+    window[requestObject.requestName] = $.ajax({
```

Comment:
Not exactly. The whole point of having an arbitrary `options` Object and using `Object.assign` is that the callee can pass in arguments `callApy` doesn't know. The `source` to `Object.assign` should simply be `options`.